### PR TITLE
FINAL: SC3-1-1-html-has-lang

### PR DESCRIPTION
**Pull request by jkodu**
- Rule based on draft - https://github.com/auto-wcag/auto-wcag/pull/85

### CODE REVIEW

File: _rules/SC3-1-1-html-has-lang.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Test Procedure
+
+### Applicability
+
+The root element of the page, if it is an `html` element.
+
+### Expectation
+
+The test target has a `lang` or `xml:lang` attribute.
```
Comment (annethyme) [inline _rules/SC3-1-1-html-has-lang.md]: Add non-empty here like in the description: 
"The test target has a [non-empty][] `lang` or `xml:lang` attribute."

File: _rules/SC3-1-1-html-has-lang.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+```html
+<html xml:lang="" lang="en">
+```
+
+```html
+<html xml:lang="en" lang="">
+```
+
```
Comment (annethyme) [inline _rules/SC3-1-1-html-has-lang.md]: I suggest adding a passed test case with a non-valid value too, to make clear that we don't test the validity of the values in this rule, e.g.:

```html
<html lang="xyz">
```

```html
<html xml:lang="xyz">
```

File: _rules/SC3-1-1-html-has-lang.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
+---
+name: Existence of HTML Lang attribute
```
Comment (WilcoFiers) [inline _rules/SC3-1-1-html-has-lang.md]: Why not "HTML has lang attribute"?

File: _rules/SC3-1-1-html-has-lang.md (hunk 1 above)
Comment (WilcoFiers) [reply _rules/SC3-1-1-html-has-lang.md]: Agreed.

File: _rules/SC3-1-1-html-has-lang.md (hunk 3 above)
Comment (jkodu) [reply _rules/SC3-1-1-html-has-lang.md]: Smart 🥇

**Comment by jkodu** [comment]
Changes done. Review again.

**Pull request merged**